src/rosdep2/platforms/gentoo.py

```diff
@@ -114,6 +115,11 @@ def get_install_command(self, resolved, interactive=True, reinstall=False, quiet
         if not atoms:
             return []
 
+        # The emerge command can be threaded.
+        cmd.append('--jobs')
+        # The next command prevents unlimited parallelization
+        cmd.append('--load-average {0}'.format(multiprocessing.cpu_count()))
```

Comment:
Before I merge this, please handle the case where `multiprocess.cpu_count()` can raise `NotImplementedError`:

https://docs.python.org/2/library/multiprocessing.html#multiprocessing.cpu_count